core/chains/evm/log/integration_test.go

```diff
@@ -1306,7 +1174,7 @@ func TestBroadcaster_ReceivesAllLogsWhenResubscribing(t *testing.T) {
 			blockHeight1: 0,
 			batch1:       []uint{1, 2},
 
-			blockHeight2:     2,
+			blockHeight2:     3,
```

Comment:
This test failure is more than just an off-by-one difference from the old `SimulateIncomingHeads` - still working on it :thinking: